app/proguard-rules.pro

```diff
@@ -1,6 +1,6 @@
 # Add project specific ProGuard rules here.
 # By default, the flags in this file are appended to flags specified
-# in /Users/ernestwong/Library/Android/sdk/tools/proguard/proguard-android.txt
+# in D:\DavidD\Projects\AndroidStudio\sdk/tools/proguard/proguard-android.txt
```

Comment:
Does this need to change ? 
If important add to a readme i have no idea what this does LOL